### ISSUE TITLE
BTable renders invalid aria-multiselectable attribute on table element

### ISSUE THREAD

**Issue by adholland** [cdmoro/bootstrap-vue-3#3234]
### Describe the bug

A select-able table has the following attribute added : -
<table aria-multiselectable="true">

This fails the AXE tool for accessibility: -
Description
Fix all of the following: ARIA attribute is not allowed: aria-multiselectable="true"[More Info (External Link)](https://dequeuniversity.com/rules/axe/4.11/aria-allowed-attr?application=axeAPI)
Element
<table id="basketTable" class="b-table-selectable table b-table table-hover table-striped table-sm" aria-busy="false" aria-multiselectable="true">

### Reproduction

https://stackblitz.com/edit/github-kucd24tm?file=src%2FApp.vue

### Used Package Manager

pnpm

**Comment by VividLemon** [comment]
I'm unsure why it appears wrong, as according to the description it appears the correct attribute https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-multiselectable thoughts?

**Comment by VividLemon** [comment]
I suppose the issue is likely is not okay for it to be on a table element. But it makes it questionable on how to convey that the rows are selectable then.

**Comment by adholland** [comment]
@VividLemon Yes it seems down to the attribute being added on the table element. So thinking the class already makes it clear it is selectable via: -
`<table class="b-table-selectable">`

possibly for accessibility then we could have an indicator for each row: -
```
<tr aria-selected="true"> 
<tr aria-selected="false">
```

not sure.
